### PR TITLE
Version upgrades

### PR DESCRIPTION
**Pull request by durbz**

### CODE REVIEW

File: ansible/roles/backend/tasks/single_Debian.yml [hunk 1]
```diff
@@ -54,18 +63,21 @@
...
-- name: Run the Teamwire backend worker container
+- name: Retrieve the Teamwire backend worker container
   when: config_done is defined
-  docker:
-    username: "{{ dockerhub_username }}"
-    password: "{{ dockerhub_password }}"
-    email: "{{ dockerhub_email }}"
+  docker_image:
+    name: "teamwire/backend:{{version|default('prod')}}"
+    state: present
```
Comment (jaydg) [inline ansible/roles/backend/tasks/single_Debian.yml]: redundant, you pull this very image before in lines 13-17

File: ansible/roles/backend/tasks/single_Debian.yml [hunk 2]
```diff
@@ -99,18 +111,21 @@
...
-- name: Run the Teamwire beat container
+- name: Retrieve the Teamwire beat container
   when: config_done is defined
-  docker:
-    username: "{{ dockerhub_username }}"
-    password: "{{ dockerhub_password }}"
-    email: "{{ dockerhub_email }}"
+  docker_image:
+    name: "teamwire/backend:{{version|default('prod')}}"
+    state: present
```
Comment (jaydg) [inline ansible/roles/backend/tasks/single_Debian.yml]: redundant, you pull this very image before in lines 13-17

**Comment by jaydg** [comment]
Please squash the three commits that change the Ansible version

**Comment by durbz** [comment]
Both fixes applied!

**Pull request merged**